### DISCUSSION TITLE
Запуск скрипта

### DISCUSSION THREAD

**Discussion by arcanium1** [Anonym-tsk/nfqws-keenetic#22] [Q&A] [answered]
Добрый день! Роутер KN-1010 на последней прошивке 4.1.7
Где-то год назад через Entware устанавливал торрент клиент. В роутер подключен внешний жесткий на ext4
Сейчас успешно подключился через порт 222, установил opkg install curl, но при шаге /bin/sh -c "$(curl -fsSL https://github.com/Anonym-tsk/nfqws-keenetic/raw/master/netinstall.sh)" получаю ошибку curl: error while loading shared libraries: libssl.so.3: cannot open shared object file: No such file or directory

Скрипты ручной установки отрабатывают, но эффекта нет.

Буду благодарен за любую помощь

**Comment by Anonym-tsk** [comment, marked as answer]
Обновить opkg и попробовать ещё раз

**Comment by arcanium1** [comment]
Спасибо! Помогло форматирование SSD и установка всего "с нуля". Работает отлично!
